paper-server/src/main/java/io/papermc/paper/configuration/PaperServerConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package io.papermc.paper.configuration;
+
+import org.spigotmc.SpigotConfig;
+
+public class PaperServerConfiguration implements ServerConfiguration {
+    @Override
+    public boolean isProxyOnlineMode() {
+        return GlobalConfiguration.get().proxies.isProxyOnlineMode();
+    }
+
+    @Override
+    public boolean isVelocityEnabled() {
+        return GlobalConfiguration.get().proxies.velocity.enabled;
+    }
+
+    @Override
+    public boolean isBungeeCordEnabled() {
+        return SpigotConfig.bungee;
+    }
+
+    @Override
+    public boolean isVelocityOnlineMode() {
+        return GlobalConfiguration.get().proxies.velocity.onlineMode;
```

Comment:
Is this actually only true if the proxy stuff is overall enabled? The new method javadocs appear to conflict with this code. The statement "is in Velocity online mode" implies, to me, that the server has velocity enabled *and* has the online-mode setting true in that section. Same for the bungee one.